backend/clubs/views.py

```diff
@@ -4861,6 +4863,39 @@ def get_queryset(self):
         )
 
 
+class WhartonCyclesView(viewsets.ModelViewSet):
+    """
+    update: Update application cycle
+    list: Get list of all application cycles
+    """
+
+    permission_classes = [WhartonApplicationPermission | IsSuperuser]
+    http_method_names = ["get", "post", "put", "patch", "delete"]
```

Comment:
Do we need `put`, `patch`, and `delete`?

Docstring only specifies update and list?

Reply:
Forgot to update the docstring, thanks for the catch! Default behavior for `post` and `delete` are used in `ModelForm` to let the user create delete cycles I believe, and `patch` is used for update.

Reply:
I meant more of:
- do we want to allow deleting?
- we should consider picking one of put/patch, they are generally considered to have similar semantics ("put" is idempotent, whereas "patch" is not always)